stripe/src/main/java/com/stripe/android/Stripe.java

```diff
@@ -1,5 +1,9 @@
 package com.stripe.android;
 
+import android.util.Log;
+
+import java.util.concurrent.Executor;
```

Comment:
following up import java.util.concurrent.Executor; it is required and was already defined.  I think an errant 'optimize imports' click in AS caused it to be added in a 2nd time in a different order. Will clean up.